tools/pythonpkg/tests/fast/pandas/test_bug5922.py

```diff
@@ -0,0 +1,10 @@
+import pandas as pd
+import duckdb
+
+df = pd.DataFrame({'col': [1,2,3]})
```

Comment:
Can you please rewrite this to be a pytest test, and add an assertion on the result?

Reply:
fixed.